c/polyjuice_utils.h

```diff
@@ -15,17 +15,23 @@
 #define debug_print(s) {}
 #define debug_print_int(prefix, value) {}
 #define debug_print_data(prefix, data, data_len) {}
-#else  /* #ifdef NO_DEBUG_LOG */
-static char debug_buffer[64 * 1024];
+#else /* NO_DEBUG_LOG */
+/* 64 KB */
+#define DEBUG_BUFFER_SIZE 65536
+static char *debug_buffer;
```

Comment:
For global, static data, let's use distinctive names, such as `g_debug_buffer`, or even in all CAPS to make sure we recognize such values immediately.